app/index.js

```diff
@@ -210,9 +210,13 @@ var BackboneGenerator = yeoman.generators.Base.extend({
       this.indexFile = this.readFileAsString(path.join(this.sourceRoot(), 'index.html'));
       this.indexFile = this.engine(this.indexFile, this);
 
-      this.indexFile = this.appendScripts(this.indexFile, 'scripts/main.js', [
-        'bower_components/requirejs/require.js'
-      ], {'data-main': 'scripts/main'});
+      var attrs = this.attributes({
+        'data-main': 'scripts/main',
+        src: 'bower_components/requirejs/require.js'
+      });
+      var scriptTag = '    <script ' + attrs + '></script>\n';
```

Comment:
if you are going to do something, might as well do it right. If its not added now, it will have to be added at some point so let's do it now and save the trouble later. Plus it doesn't seem to be a lot more to make it happen. Also, thank you for your help with this @marian-r